packages/api-plugin-fulfillment/src/mutations/createFulfillmentMethod.test.js

```diff
@@ -0,0 +1,72 @@
+import mockCollection from "@reactioncommerce/api-utils/tests/mockCollection.js";
+import mockContext from "@reactioncommerce/api-utils/tests/mockContext.js";
+import createFulfillmentMethodMutation from "./createFulfillmentMethod.js";
+
+mockContext.validatePermissions = jest.fn().mockName("validatePermissions");
+mockContext.collections.Fulfillment = mockCollection("Fulfillment");
+mockContext.validatePermissions.mockReturnValueOnce(Promise.resolve(null));
+
+test("throws if required fields are not supplied", async () => {
+  const fulfillmentMethodInput = {
+    method: {
+      shopId: "SHOP_ID",
+      cost: 99,
+      fulfillmentMethod: "flatRates",
+      displayMessageMethod: "Sample display message"
+    },
+    fulfillmentTypeId: "fulfillment123",
+    shopId: "SHOP_ID"
+  };
+
+  await expect(createFulfillmentMethodMutation(mockContext, fulfillmentMethodInput)).rejects.toThrowErrorMatchingSnapshot();
```

Comment:
Since there are no other opinions yet, I shall move this comment to the new PR (#6610) and resolve this comment.
@zenweasel Please let me know if this can be considered resolved

Reply:
Since there are no disagreements then my original comment stands. Snapshots are brittle and a very jest specific thing and I don't think we should introduce any new ones

Reply:
@zenweasel 
Sure. I have already copied over this comment to the new PR and shall make the changes in that branch. I will be resolving this comment.